app/Resources/views/composer/install.html.twig

```diff
@@ -0,0 +1,74 @@
+{% macro command(command) %}
+    <div class="orange">
+        <p>&#36; {{ command }}</p>
+    </div>
+{% endmacro %}
+
+{% import _self as Helper %}
+
+<div class="window">
+    <div class="titlebar">
+        <div class="buttons">
+            <div class="close">
+                <a class="closebutton" href="#"><span><strong>x</strong></span></a>
+            </div>
+            <div class="minimize">
+                <a class="minimizebutton" href="#"><span><strong>&ndash;</strong></span></a>
+            </div>
+            <div class="zoom">
+                <a class="zoombutton" href="#"><span><strong>+</strong></span></a>
+            </div>
+        </div>
+        ezplatform@eZ-Systems: ~/www
+    </div>
+    <div class="content">
+        {% set dbhost = "127.0.0.1" %}
+        {% set dbport = 3333 %}
+        {% set dbname = 'ezplatform' %}
+        {% set dbpassword = 'ezplatform' %}
+
+        <div class="help">
+            <p># Install composer</p>
+            <div class="orange">
+                {{ Helper.command("curl -sS https://getcomposer.org/installer | php") }}
+            </div>
+        </div>
+        <div class="help">
+            <p># If you do not have a MySQL server ready, you can create one quickly using Docker.</p>
+            <p># Note that we are using here the TCP Port <em class="green">{{ dbport }}</em> of the host and
+                <em class="green">ezplatform</em> as root's password for the MariaDB server container.</p>
+            {{ Helper.command("docker run -p "~dbport~":3306 --name ezdbcontainer -e MYSQL_ROOT_PASSWORD="~dbpassword~" mariadb:10.0") }}
+        </div>
+        <div class="help">
+            <p># Install eZ Platform</p>
+            {{ Helper.command("php -d memory_limit=-1 composer.phar create-project --no-dev ezsystems/ezplatform") }}
+        </div>
+        <div class="help">
+            <p># Depending on your platform (i.e quick test on Mac OS X) you might get a dependency error on
+                <em class="green">ext-intl</em>
+            </p>
+            <p># If so, just for testing you can ignore it using the option
+                <em class="green">--ignore-platform-reqs</em></p>
+            <p># Note that you may also need to set up correctly your date.timezone configuration for your php.</p>
+            {{ Helper.command("php -d memory_limit=-1 composer.phar create-project --no-dev --ignore-platform-reqs ezsystems/ezplatform") }}
+        </div>
+
+        <div class="help">
+            <p># Create the database and import clean data</p>
+            {{ Helper.command("mysql -u root -p"~dbpassword~" -P "~dbport~" -h "~dbhost~" -e \"CREATE DATABASE "~dbname~" CHARACTER SET utf8;\"") }}
```

Comment:
would be better if you could use single quotation marks for Twig helpers in this template

there is also 
```twig
{{ Helper.command('php app/console doctrine:database:create' }}
```
command which can be used to create a new database using `parameters.yml` settings